### DISCUSSION TITLE
The UI display

### DISCUSSION THREAD

**Discussion by r91085** [iterate-ch/cyberduck#15385] [Q&A] [answered]
The UI elements display is not complete when the system font is bigger than normal. I can't know if I miss some setttings or information. Can I control the ui windows size ?

![ScreenClip](https://github.com/iterate-ch/cyberduck/assets/37505035/d2a901fa-6140-4772-8e2a-362576fdcb81)
![ScreenClip](https://github.com/iterate-ch/cyberduck/assets/37505035/53d16978-71e3-4bf4-9564-cd9efd324187)

**Comment by dkocher** [comment, marked as answer]
This is tracked in #8961
